esphome/core/entity_base.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <string>
```

Comment:
```suggestion
#include <string>
#include <cstdint>
```

Nit: `uint32_t` from that header is used here, and not explicitly included (probably all call sites have cstdint imports themselves before including this)

Reply:
Done!